### PR TITLE
pipewire.md: warn about XDG_RUNTIME_DIR

### PR DESCRIPTION
**Pull request by st3r4g**
Particularly nasty because no clear error messages are shown. Tripped up quite some users on IRC and [reddit](https://old.reddit.com/r/voidlinux/comments/nq3suy/pipewirepulse_and_void_linux_and_a_pile_of_other/)

### CODE REVIEW

**Comment by st3r4g** [comment]
could be also added in the `pulseaudio-replacement` subsection, but maybe adds noise

**Comment by st3r4g** [comment]
Also, maybe the explanation on how to setup XDG_RUNTIME_DIR should be moved to to the session management page.

File: src/config/media/pipewire.md [hunk 1]
```diff
@@ -64,3 +64,9 @@ on glibc-based systems:
 # echo "/usr/lib/pipewire-0.3/jack" > /etc/ld.so.conf.d/pipewire-jack.conf
 # ldconfig
 ```
+
+## Troubleshooting
+
+The Pulseaudio replacement requires
+[`XDG_RUNTIME_DIR`](../graphical-session/wayland.html#configuration) to work
```
Comment (ericonr) [inline src/config/media/pipewire.md:71]: "the [XDG_RUNTIME_DIR](link) environment variable"

File: src/config/session-management.md [hunk 2]
```diff
@@ -52,3 +52,12 @@ non-root users to be able to access the seatd session, add them to the `_seatd`
...
 Note that, unlike elogind, seatd doesn't do anything besides managing seats.
+
+## XDG_RUNTIME_DIR
+
+Install [elogind](#elogind) as your session manager to automatically setup
+`XDG_RUNTIME_DIR`.
+
+Alternatively, manually set the environment variable through the shell. Make
+sure to create a dedicated user directory and set its permissions to `700`. A
+good default location is `/run/user/$(id -u)`.
```
Comment (ericonr) [inline src/config/session-management.md:63]: The new section is missing an introduction explaining that it is an environment variable. Maybe link an official page about XDG spec, if such a thing exists.

File: src/config/session-management.md [hunk 3]
```diff
@@ -52,3 +52,12 @@ non-root users to be able to access the seatd session, add them to the `_seatd`
 group.
 
 Note that, unlike elogind, seatd doesn't do anything besides managing seats.
+
+## XDG_RUNTIME_DIR
+
+Install [elogind](#elogind) as your session manager to automatically setup
+`XDG_RUNTIME_DIR`.
```
Comment (ericonr) [inline src/config/session-management.md:59]: I think it should be "set up"

File: src/config/graphical-session/wayland.md [hunk 4]
```diff
@@ -80,15 +80,9 @@ compositors, and is installed as a dependency for most of them. Its package is
...
-The Wayland library uses the `XDG_RUNTIME_DIR` environment variable to determine
-the directory for the Wayland socket.
-
-Install `elogind` as your [session manager](../session-management.md) to
-automatically setup `XDG_RUNTIME_DIR`.
-
-Alternatively, manually set the environment variable through the shell. Make
-sure to create a dedicated user directory and set its permissions to `700`. A
-good default location is `/run/user/$(id -u)`.
+The Wayland library uses the
```
Comment (ericonr) [inline src/config/graphical-session/wayland.md:83]: I guess we can improve the text to say "requires" instead of "uses", since it's mandatory.

**Pull request merged**